### PR TITLE
Download filter and badge added

### PR DESCRIPTION
**Pull request by abhijeetChawla**
Added the download filter to the gallery
added the download badge to the manga card

### CODE REVIEW

File: src/util/useLibraryOptions.ts [hunk 1]
```diff
@@ -33,6 +39,6 @@ export default function useLibraryOptions(): IUseLibraryOptions {
     // eslint-disable-next-line eqeqeq
     const active = !(unread == undefined);
```
Comment (ntbm) [inline src/util/useLibraryOptions.ts:40]: add the active condition here as well.

**Comment by AriaMoradi** [comment]
why close?

File: src/util/useLibraryOptions.ts (hunk 1 above)
Comment (abhijeetChawla) [reply src/util/useLibraryOptions.ts:40]: I will do that, but i am closing this pull request. I screwed up with git so i am closing this and making another pull request. I am so sorry

**Comment by abhijeetChawla** [comment]
git screw up, i think

**Pull request closed without merging**